x/auth/keeper/grpc_query_test.go

```diff
@@ -192,83 +194,114 @@ func (suite *KeeperTestSuite) TestGRPCQueryParameters() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestGRPCQueryModuleAccounts() {
```

Comment:
@cyberbono3 I just noticed this test doesn't exist anymore. Is there a reason you deleted it?

Reply:
I accidentally  removed it. Sorry about that.